src/game/client/components/menus.cpp

```diff
@@ -1697,7 +1697,9 @@ int CMenus::Render()
 			pButtonText = Localize("Abort");
 			if(Client()->MapDownloadTotalsize() > 0)
 			{
-				pTitle = Localize("Downloading map");
+				char aBuf[512];
+				str_format(aBuf, sizeof(aBuf), "%s: %s", Localize("Downloading map"), Client()->MapDownloadName());
+				pTitle = aBuf;
```

Comment:
`pTitle` is referenced later on, but `aBuf` is tied to this scope. The `aBuf` declaration should probably move up to the top of the function.